packages/plugins/src/api/index.js

```diff
@@ -105,6 +108,12 @@ export function registerPlugin( name, settings ) {
 		);
 		return null;
 	}
+	if ( settings.priority && typeof settings.priority !== 'number' ) {
```

Comment:
Extracting `priority` to a variable would improve a little bit readability.

Reply:
What if priority is falsey?

Reply:
we can use optional-chaining: https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining :)

Reply:
@nicolad What I meant is that this code doesn't work when using an empty string as the priority:

```js
var settings = { priority: '' };
(settings.priority && typeof settings.priority !== 'number') || 'no error'
```

will return 'no error'

Reply:
I see, then:
```
const priority = settings.priority && settings.priority.length && typeof settings.priority !== 'number';
```
if `priority` property is always present in `settings` object, than first check is not needed.

Reply:
@nicolad @swissspidy Thanks for the great feedback here! I have made some changes to this section and added some unit tests.